tasks/StoringGroceries.tex

```diff
@@ -52,9 +56,11 @@ \subsection*{Procedure}
 		\begin{itemize}
 			\item whether the cabinet door should be closed
 			\item whether a heavy and/or a tiny object should be used in the test
+			\item whether the robot plans to pour the milk
```

Comment:
what about cereal?

Reply:
As this is a main goal the cereal should be always present